src/maintainer/conda_forge_yml.rst

```diff
@@ -79,6 +79,24 @@ mapping for Azure-specific configuration options. For example:
       # built packages) as an Azure pipeline artifact that can be downloaded
       store_build_artifacts: False
 
+.. _self-hosted_azure-config:
```

Comment:
Shall i remove this line too?

Reply:
Good question. Actually I think we should keep it for referring to the config below in the intro.